activities/activity_anna_petrasova.py

```diff
@@ -2,15 +2,12 @@
 
 """
 Instructions
-
 - Functions intended to run for each scan
   need to be named run_xxxxx
-
 - Do not modify the parameters of the run_xxx function
   unless you know what you are doing
   see optional parameters:
   https://github.com/tangible-landscape/grass-tangible-landscape/wiki/Running-analyses-and-developing-workflows#python-workflows
-
```

Comment:
Can you add back the empty lines here. Then this should disappear from the PR.

Reply:
I added the lines back in.